src/components/pages/RoleSignup/Applications/Mentee.js

```diff
@@ -43,170 +49,257 @@ const Mentee = () => {
-            }}
-          >
-            <Input placeholder="Your Name" />
-          </Form.Item>
-        </div>
-
-        <div className="lives_in_us">
-          <h3>Do you live in the U.S?*</h3>
-          <Radio.Group value={formValues.lives_in_us}>
-            <Radio
-              onChange={evt => {
-                inputChange('lives_in_us', evt.target.value);
-              }}
-              value="Yes"
-            >
-              Yes
-            </Radio>
-            <Radio
-              onChange={evt => {
-                inputChange('live_in_us', evt.target.value);
-              }}
-              value="No"
-            >
-              No
-            </Radio>
-          </Radio.Group>
-        </div>
-
-        <div className="formerly_incarcerated">
-          <h3>Are you formerly incarcerated/have a felony?*</h3>
-          <Radio.Group value={formValues.formerly_incarcerated}>
-            <Radio
-              onChange={evt => {
-                inputChange('formerly_incarcerated', evt.target.value);
-              }}
-              value="Yes"
-            >
-              Yes
-            </Radio>
-            <Radio
-              onChange={evt => {
-                inputChange('formerly_incarcerated', evt.target.value);
-              }}
-              value="No"
-            >
-              No
-            </Radio>
-          </Radio.Group>
-        </div>
-
-        <div className="list_convictions">
-          <h3>Please list your convictions if comfortable</h3>
-          <Form.Item
-            type="text"
-            name="list_convictions"
-            value={formValues.list_convictions}
-            onChange={evt => {
-              inputChange('list_convictions', evt.target.value);
-            }}
-          >
-            <Input placeholder="Your answer" />
-          </Form.Item>
-        </div>
+    <div>
+      <div className="breadcrumbs">
+        <Breadcrumb>
+          <Breadcrumb.Item href="/login">
+            <HomeOutlined />
+          </Breadcrumb.Item>
+          <Breadcrumb.Item href="/signup">
+            <IdcardOutlined />
+            <span>Signup</span>
+          </Breadcrumb.Item>
+          <Breadcrumb.Item>
+            <ReconciliationOutlined />
+            <span>Mentee Application</span>
+          </Breadcrumb.Item>
+        </Breadcrumb>
+      </div>
+      <div className="application">
+        <Form onFinish={formSubmit}>
+          <div className="signUpForm">
+            <h1> Mentee Application </h1>
+            <div className="questions">
+              <div className="infoDiv">
+                <h3>Please fill out your user information</h3>
+                <br />
+                <div className="name">
+                  <div className="titleContainer">
+                    <h3>Full Name*</h3>
+                  </div>
+                  <Form.Item
+                    type="text"
+                    name="name"
+                    rules={[
```

Comment:
I did not know Ant Design had the rules feature in the forms, this is a great feature for our forms.

Reply:
Oh wow, this is fantastic to know, I like this!